corehq/apps/toggle_ui/templates/toggle/edit_flag.html

```diff
@@ -138,21 +136,21 @@
         </div>
         <hr/>
         <!-- ko foreach: items -->
-        <div class="row">
-            <div class="span1">
-                <a href="#" class="btn btn-danger" data-bind="click: $parent.removeItem"><i class="icon-trash"></i></a>
+        <div class="row" style="margin: 5px 0">
```

Comment:
the delete button and the input should both be inline in the same column. Also, let's stay away from using inline styles